docs/leap_sapi/sapi_rest.rst

```diff
@@ -1024,11 +1024,13 @@ The request should contain no body.
 List Problems
 -------------
 
-To retrieve a list of previously submitted problems, send an HTTP GET request to
-``problems``.
+To retrieve a list of problems, send an HTTP GET request to ``problems``.
 
 The request should contain no body.
 
+.. note:: Up to 1000 of a user's newest problem submissions are stored
+    and accessible in the Leap service for up to 365 days.
+
```

Comment:
Same comment in [docs/leap_sapi/admin.rst](https://github.com/dwavesystems/dwave-ocean-sdk/pull/334/files#diff-9098cce8f62fcbe9a6e248822c45ad67bef4d5073a3955846ef887e07e6e4d63).

Here, given the low number of users creating their own client, the benefit is even lower.